src/nist.gov/SP800-53/rev5/xml/NIST_SP-800-53_rev5_catalog.xml

```diff
@@ -1,14 +1,17 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <?xml-model schematypens="http://www.w3.org/2001/XMLSchema" type="application/xml" href="https://github.com/usnistgov/OSCAL/releases/download/v1.1.1/oscal_complete_schema.xsd"?>
-<catalog xmlns="http://csrc.nist.gov/ns/oscal/1.0" uuid="0e9e8558-4d93-4a7f-8c87-34d4d4f9e7bd">
+<catalog xmlns="http://csrc.nist.gov/ns/oscal/1.0" uuid="946e5386-e7b1-48bd-9bfd-1f5bfa59b09b">
   <metadata>
     <title>Electronic Version of NIST SP 800-53 Rev 5 Controls and SP 800-53A Rev 5 Assessment Procedures</title>
-    <last-modified>2023-10-12T00:00:00.000000-04:00</last-modified>
+    <last-modified>2023-11-14T00:00:00.000000-04:00</last-modified>
     <version>5.1.1</version>
     <oscal-version>1.1.1</oscal-version>
     <prop name="keywords" value="Assessment, assessment plan, assurance, availability, computer security, confidentiality, control, control assessment, cybersecurity, FISMA, information security, information system, integrity, personally identifiable information, OSCAL, Open Security Controls Assessment Language, Privacy Act, privacy controls, privacy functions, privacy requirements, Risk Management Framework, security controls, security functions, security requirements, system, system security"/>
     <link rel="alternate" href="#c3397cc9-83c6-4459-adb2-836739dc1b94"/>
     <link rel="canonical" href="#f7cf488d-bc64-4a91-a994-810e153ee481"/>
+    <link rel="latest-version" href="#5a5ffcb9-2272-484e-8d47-4483a0585dec"/>    
+    <link rel="predecessor-version" href="#5a5ffcb9-2272-484e-8d47-4483a0585dec"/>
+    <link rel="successor-version" href="#5a5ffcb9-2272-484e-8d47-4483a0585dec"/>
```

Comment:
I may be missing the correct branch or location, but where is the `5a5ff` document referred to here?  It is marked as the predecessor, successor and latest?

Reply:
This points to the [oscal-content releases page](https://github.com/usnistgov/oscal-content/releases/) in the `back-matter` `resources` (see [this commit since the whole file diff in GitHub won't render](https://github.com/usnistgov/oscal-content/commit/21160e6e3a1d7d7e20bac978b3c4967eeeea461c#diff-f22c17cf618814bac44a430c73fa1e560e5a2c82aef54a9bdf28a18d80958144R83622-R83625)), but could be adjusted to point to other URLs, but they would need frequent pruning or changes to be specific about what is latest, the successor, and predecessor. I would do this as a compromise because pointing to specific versions on commit or tags in this link construct, or as requested in `revisions` will need constant tending or care. If managed through git SHA1 commit IDs and permalinks, you run into a Schroedinger's commit situation where you want to specifically link to a commit where SHA1 might change too (especially for `@rel="latest"` and `@rel="successor"`, but that is an edge case).

Does that make sense? Would you prefer something different?

Reply:
I am not sure how we can explain that "predecessor-version" and "successor-version" are identical. . I understand one of them to be identical with the"latest-version", either because it is the first version ever (no other predecessor) or because we will never have another future version (it sunsets).  Can any of those not be included?

Reply:
I'm now tracking what AJ is outlining here.

In reading through the reference, I see that the `rel` attributes used in links above are from the constraints outlined for [links under revisions](https://pages.nist.gov/OSCAL-Reference/models/v1.1.1/catalog/json-reference/#/catalog/metadata/revisions).  If we move these links under revisions, we can add the version information that would provide what we are missing.

```
revisions [0 or 1]: 
    title [0 or 1]: markup-line,
    published [0 or 1]: date-time-with-timezone,
    last-modified [0 or 1]: date-time-with-timezone,
    version [1]: string,
    oscal-version [0 or 1]: string,
    props [0 or 1]
    links [0 or 1]
    remarks
```
- title: can provide some descriptive title...
- version: this is where we can provide and update sequence number, e.g. u1, u2 (or just a number)
- oscal-version: should we update to a new version of oscal, where the content is the same, this would be helpful historically
- remarks: can describe the revision (Incorporates v.... of ..., corrects misspelling of..., etc)
- published: maybe?

One question remaining is if the versions updated in each revision will change the document oscal-version and version.  If we do, the version in the revisions likely needs to be in the format of 5.1.1+u1, 5.1.1+u2.

In this case, we wouldn't have to depend solely upon hashes.  We might want to create an ADR on how we intend to version, understanding that others might be looking for the same guidance, or understanding.  If this is the case, we can take the PR and update it after the ADR.  I'm willing to get that started so we can work it out there.

Reply:
Oh, it seems I misinterpreted the Metaschema source (I thought this might incorrect due to https://github.com/usnistgov/OSCAL-Reference/issues/6, it is actually correct and the constraint is not any link but in the `revision/link`, apologies. That said, now we walk ourselves into keeping a changelog in the document itself. I will adjust it for this first time, the problem becomes worse for subsequent edits. (To my knowledge this is the last one ever for this approach, so maybe not a big deal).

Reply:
I move to the recommendation by Chris to correctly use `revisions` with my relocated rel and using `version-history` (in 1.1.1 from the RFC and better addressing @iMichaela's complaints) in https://github.com/usnistgov/oscal-content/commit/e6bfb87252ba1b49d609371333122a0778074e64.

Reply:
I like this idea too. I was going to suggest `revisions` but didn't want to create extra work. Thanks @Compton-NIST